package.json

```diff
@@ -219,11 +219,10 @@
     "columnify": "^1.5.2",
     "common-tags": "^1.7.2",
     "denymount": "^2.3.0",
-    "docker-modem": "^2.1.4",
-    "docker-progress": "^4.0.1",
+    "docker-modem": "^3.0.0",
+    "docker-progress": "^5.0.0",
     "docker-qemu-transpose": "^1.1.1",
-    "docker-toolbelt": "^3.3.8",
-    "dockerode": "^2.5.8",
+    "dockerode": "^3.3.0",
```

Comment:
My testing so far did not reveal any problems with `dockerode` v3. However, the CLI is only partially using it, as a few dependencies still require `dockerode` v2:
```
$ npm list
...
├─┬ balena-sync@11.0.2
│ ├─┬ docker-toolbelt@3.3.8
│ │ ├─┬ dockerode@2.5.8

├─┬ livepush@3.5.0
│ ├─┬ dockerode@2.5.8

├─┬ resin-multibuild@4.11.0
│ ├─┬ dockerode@2.5.8

├─┬ resin-multibuild@4.11.0
│ ├─┬ resin-docker-build@1.1.6
│ │ ├─┬ dockerode@2.5.8
```

`resin-multibuild` still declares a dependency on `dockerode` v2, but it mostly uses a `dockerode` instance injected as an argument in several functions, which after this PR will be `dockerode` v3 for `balena-cli`. The balenaCloud builder will still use `dockerode` v2 as long as that's the version it injects. I have tested this PR quite a bit (and automated tests cover it too). We'll keep an eye on it over time, and gradually move components from v2 to v3.